src/test/java/core/basesyntax/strategy/impl/PurchaseTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.strategy.impl;
+
+import static core.basesyntax.model.FruitTransaction.Operation;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.FruitsStorage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseTest {
+    private static final FruitTransaction CORRECT_TRANSACTION
+            = new FruitTransaction(Operation.PURCHASE, "banana", 20);
+    private static final FruitTransaction INCORRECT_MORE_THAN_PRESENT
+            = new FruitTransaction(Operation.PURCHASE, "banana", 150);
+    private static final FruitTransaction INCORRECT_ENOTHER_OPER
```

Comment:
```suggestion
    private static final FruitTransaction INCORRECT_ENOTHER_OPERATION
```
 не використовуй скорочень за можливості